src/v8/v8_script.rs

```diff
@@ -59,11 +122,41 @@ impl V8PersistedScript {
                 self.inner_persisted_script,
             )
         };
+        let code = ScriptCode::new(&self.code, isolate_scope);
         V8LocalScript {
             inner_script,
+            code,
```

Comment:
I guess this is needed for the recompile? If we conclude that we do not need the recompile we should remember to drop it.